### PR TITLE
added page with working with firewalls, python examples

### PR DESCRIPTION
**Pull request by edsonarios**
Issue link: [https://github.com/softlayer/githubio_source/issues/523]

### CODE REVIEW

File: content/python/firewalls.md [hunk 1]
```diff
@@ -0,0 +1,454 @@
...
+    def get_firewall_ip_address(self, primary_ip_address):
+        vlan = self.client.call('SoftLayer_Network_Vlan', 'getVlanForIpAddress', primary_ip_address)
+        self.env.fout(formatting.iter_to_table(vlan))
+        firewall = self.client.call('SoftLayer_Network_Vlan', 'getFirewallProtectableIpAddresses', id=vlan['id'], limit=10)
+        self.env.fout(formatting.iter_to_table(firewall))
+
+```
+
+## Edit Standard Rules
+Edit Standard Rule`.
```
Comment (BrianSantivanez) [inline content/python/firewalls.md:202]: Remove the single quote before the end point, after that it looks good to me

**Review by allmightyspiff — CHANGES_REQUESTED** [review]
Looks good, but please also remove the old examples pages as well.

**Comment by edsonarios** [comment]
> Looks good, but please also remove the old examples pages as well.

The old examples were removed

File: content/python/firewalls.md (hunk 1 above)
Comment (edsonarios) [reply content/python/firewalls.md]: The single quote was removed

**Review by allmightyspiff — APPROVED** [review]

**Pull request merged**